### PR TITLE
initial release of http api

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/initial-http-api-is-released.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Title: Initial HTTP API is released
```
Comment (trel) [inline content/posts/initial-http-api-is-released.md:1]: Initial release of the iRODS HTTP API

File: content/posts/initial-http-api-is-released.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Date: 2023-11-08 20:00
+Author: Kory Draughn
+Slug: initial-http-api-is-released
```
Comment (trel) [inline content/posts/initial-http-api-is-released.md:4]: same

File: content/posts/initial-http-api-is-released.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+The iRODS Consortium is pleased to announce the first release of the [iRODS HTTP API](https://github.com/irods/irods_client_http_api).
+
+It is released today as v0.1.0.
+
+It is an experimental redesign of the iRODS C++ REST API originally released in 2021. The primary goal of the project is to make it easy and safe for developers to interact with an iRODS zone. This falls right in line with the goals outlined in the [Consortium's roadmap for 2023](https://irods.org/roadmap).
+
+Users of the iRODS C++ REST API will find the iRODS HTTP API simpler to work with and more equipped for building rich applications and managing complex workloads. It also has support for OpenID Connect.
+
+A [Docker image](https://hub.docker.com/r/irods/irods_http_api) has been published on hub.docker.com for those interested in giving it a try.
```
Comment (trel) [inline content/posts/initial-http-api-is-released.md:16]: update
"has been published on hub.docker.com for those interested in giving it a try."
to 
"has also been published on Docker Hub for ease of deployment."

File: content/posts/initial-http-api-is-released.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+Watch the Consortium's iRODS User Group Meeting 2023 talk or visit the [GitHub repository](https://github.com/irods/irods_client_http_api) to learn more.
```
Comment (trel) [inline content/posts/initial-http-api-is-released.md:18]: link
Consortium's iRODS User Group Meeting 2023 talk
with
https://www.youtube.com/watch?v=9iF5nHTvO0w&list=PL29FhEN41mZPWLhE4CY4AvWA1vXygxoXx

Comment (korydraughn) [reply content/posts/initial-http-api-is-released.md:18]: Should I drop the `<iframe>` then?

Comment (trel) [reply content/posts/initial-http-api-is-released.md:18]: hmm.    no?    one is a direct link to the video in the playlist.   and then it's also embedded?

Comment (korydraughn) [reply content/posts/initial-http-api-is-released.md:18]: Gotcha. Pushing updates now.

File: content/posts/initial-http-api-is-released.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/initial-http-api-is-released.md]: Done.

File: content/posts/initial-http-api-is-released.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/initial-http-api-is-released.md]: Done.

File: content/posts/initial-http-api-is-released.md (hunk 3 above)
Comment (korydraughn) [reply content/posts/initial-http-api-is-released.md]: Done.

File: content/posts/initial-http-api-is-released.md (hunk 4 above)
Comment (korydraughn) [reply content/posts/initial-http-api-is-released.md]: Done.

**Comment by trel** [comment]
oh, please also update the filename itself to match the title

**Comment by korydraughn** [comment]
Renamed the file.

Are we ready for squashing?

**Comment by trel** [comment]
yes please squash

**Comment by korydraughn** [comment]
Squashed it!

**Comment by trel** [comment]
sold.

**Pull request merged**